runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/metrics/reader/MetricRecord.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2021-2023 Aklivity Inc.
+ *
+ * Aklivity licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.aklivity.zilla.runtime.engine.metrics.reader;
+
+public interface MetricRecord
+{
+    long namespacedBindingId();
+
+    int bindingId();
+
+    String namespaceName();
+
+    String bindingName();
+
+    String metricName();
```

Comment:
Let's go with `namespace()`, `binding()` and `metric()`, dropping the `Name` suffix.

Why do we need both `bindingId()` and `namespacedBindingId()`?
Can we make do with just `namespacedBindingId()` and call it `bindingId()`? 😄

Reply:
done